### DISCUSSION TITLE
特斯拉人形機器人原型亮相！馬斯克：目前「沒腦」

### DISCUSSION THREAD

**Discussion by Yehchenghan** [1101-GS4719-C/Python-Community-GS4719-C#737] [General]
雖然現在還沒辦法有自己思考的機器人，但能取代人類的項目越來越多了。
[原文網址](https://tw.news.yahoo.com/%E7%89%B9%E6%96%AF%E6%8B%89%E4%BA%BA%E5%BD%A2%E6%A9%9F%E5%99%A8%E4%BA%BA%E5%8E%9F%E5%9E%8B%E4%BA%AE%E7%9B%B8-%E9%A6%AC%E6%96%AF%E5%85%8B-%E7%9B%AE%E5%89%8D-%E6%B2%92%E8%85%A6-111248157.html)

**Comment by danwan1027** [comment]
人類目前的許多工作未來終將會被機器人取代

**Reply by Yunhans** [reply]
所以要多學一些技能才行

**Comment by AnAnistired** [comment]
機器人取代人類工作，就跟18、19世紀工業革命一樣，能夠推進人類文明進步

**Comment by williamyen0623** [comment]
一定要當上做那顆腦的人

**Comment by evanchou1204** [comment]
金氏世界紀錄有全球速度最快的機器人這個項目也是很問號？？？為什麼需要機器人來跑百米？
